src/components/display/fewlines/Input.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+export const Input = styled.input`
+  background: ${({ theme }) => theme.colors.backgroundContrast};
+  border: 0.1rem solid #a1a4b1;
+  border-radius: ${({ theme }) => theme.radii[0]};
+  height: 4rem;
+
+  ::placeholder {
+    color: #8b90a0;
+    font-size: 1.4rem;
```

Comment:
Font size should come from the theme. Don't forget we can change values from the theme if needed.

I think your next task should be related to refactoring the theme, following the mockups :).

Reply:
OK fixed d40bcb1